unix.c

```diff
@@ -158,15 +158,19 @@ int main(int argc, char *argv[])
 		pw = getpwnam(global.conf->user);
 		if (!pw) {
 			log_message(LOGLVL_ERROR, "Failed to look up user %s.", global.conf->user);
+			return(EXIT_FAILURE);
```

Comment:
Why EXIT_FAILURE here and 1 for the other cases?

Reply:
I used what is used in the remaining codebase. In general all hardcoded values should be replaced, but even if I did this for this PR here there's a lot left to fix. But I can change that if you like

Reply:
I'm not expecting this PR to update all other places (in fact, that would be a distraction), but it seems inconsistent even for new return() calls that it introduces.

Reply:
okay :)